site/contents/write_ups/2022/12_01_22-writeup.md

```diff
@@ -0,0 +1,93 @@
+
+Huw's resolution is to give outreach talks to schools talking about Data ethics and Weapons of Maths Destruction. If the talks inspire the next generation of STEM specialists to be more data ethically conscious, this is brilliant, but even if this does not happen, it is enough to start conversations about this sort of topic. Additionally, Huw has been beating around the bush about writing some data ethics related blog posts. Time to get to it.
+
+Alastair was think has started thinking about ways he can encourage students and other learners to consider the ethical context of their work. Statistical mishandling is often forced error by the environment of academia that we're in and it is very easy for academic fraud to creep in. How do we teach and warn against this in the context of academia in a practical way?
+
+Our very own Natalie wants to publish more Data Ethics content (e.g. how to run your Data Ethics room, Data hazards), aiming to doing even more to make the formula we have at Data Ethics Club replicatable. Natalie also wants us to shy away less from more technical reading material at Data Ethics club, potentially suggesting some relevant reading. Natalie is also considering running teaching sessions for undergraduates on Data Ethics. 
+
+Vanessa hopes to run some practical sessions during Data Ethics meetings. Vanessa plans on writing a hate piece on the Metaverse, a piece we are all very excited for. Vanessa is also going to look into [Is debiasing bad?](https://edri.org/our-work/if-ai-is-the-problem-is-debiasing-the-solution/), understand the arguments and potentially make use of them in her thesis if relevant. Roman wants to write an article about ethics education in computer science.
+
+Nina wants to publish a [Data Hazards](https://very-good-science.github.io/data-hazards/) analysis of her projects on the Open Science Framework.
+
+## Balancing saltiness and sweetness
+
+Ola's resolution is to, as she put it, balance saltiness and sweetness in her approach to academic reading. This is not just because of politeness. Rather than being bitter and defensive against academic reading, is there something to do be done to make the reading about the author and not your own ego? How do we learn to love reading again and not see it as a chore? 
+
+Ola will be trying to take a slightly more positive outlook on things whilst still being thorough. Paper reviewers could benefit from this too! This idea resonated with lots of members of the group, who missed the days when they could genuinely read for joy instead of just for work. Unfortunately PhD students are just being optimised for picking out "what can I get out of this?" from any source they read from
```

Comment:
```suggestion
Ola will be trying to take a slightly more positive outlook on things whilst still being thorough. Paper reviewers could benefit from this too! This idea resonated with lots of members of the group, who missed the days when they could genuinely read for joy instead of just for work. Unfortunately PhD students are just being optimised for picking out "what can I get out of this?" from any source they read from.
```

Sad and true...